animatch/SingUp/SingUpView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  SingUpView.swift
+//  animatch
+//
+//  Created by Ксения Панкратова on 23.11.2024.
+//
+
+import SwiftUI
+
+struct SingUpView: View {
+    @State private var inputEmail = ""
+    @State private var inputPasswordFirst = ""
+    @State private var inputPasswordSecond = ""
+
+    var body: some View {
+        VStack {
+            ZStack {
+                backgroundLineGradient
+
+                VStack(spacing: 0) {
+                    Logo
+                    formsContainer
+                    buttonsContainer
+                    Spacer()
+                    footer
+                }
+            }
+        }
+        .ignoresSafeArea()
+    }
+}
+
+// MARK: - UI Subviews
+
+private extension SingUpView {
+
+    var backgroundLineGradient: some View {
+        LinearGradient(
+            gradient: Gradient(
+                colors: [
+                    .editProfLightGrad,
+                    .editProfDarkGrad
+                ]
+            ),
+            startPoint: .top,
+            endPoint: .center
+        )
+        .ignoresSafeArea()
+    }
+
+    var formsContainer: some View {
+        VStack(spacing: 0) {
+            Text(Constants.formsContainerTitle)
+                .foregroundStyle(Color.editProfWhite)
+                .font(Font.custom("Roboto", size: 32))
+                .padding(.bottom, 34)
+
+            NGGTextField(title: Constants.nameTextFieldPlaceholder, text: $inputEmail)
+            NGGSecureField(Constants.passwordTextFieldPlaceholder, text: $inputPasswordFirst)
+            NGGSecureField(Constants.passwordrepeatTextFieldPlaceholder, text: $inputPasswordSecond)
+            
+        }
+        .padding(.top, 60)
+    }
+
+    var Logo: some View {
+        Image(.logo)
+            .resizable()
+            .frame(width: 209, height: 69)
+            .padding(.top, 150)
+    }
+
+    var buttonsContainer: some View {
+        VStack(spacing: 0) {
+            NGGButton(Constants.continueButtonTitle) {
+                // TODO: IOS-15: Добавить логику обработки нажатий
+                print("[DEBUG]: Продолжить")
+            }
+        }
+        .padding(.bottom, 80)
+    }
+    
+    var footer: some View {
+        VStack{
+            Divider()
+                .overlay(.white)
+                .frame(width: 320)
+            
+            HStack {
+                Text(Constants.haveAccountText)
+                    .foregroundStyle(Color.white)
+                    .font(Font.custom("Roboto", size: 16))
+            
+                Button {
+                    // TODO: IOS-15: Добавить логику обработки нажатий
+                    print("[DEBUG]: Вход")
+                }label: {
+                    Text(Constants.singupButton)
+                        .underline()
+                        .foregroundColor(.editProfPurple)
+                }
+            }
+            
+        }
+        .padding(.bottom, 130)
+    }
```

Comment:
Тут опять же проблемка пробелами и отступами

```suggestion
    var footer: some View {
        VStack {
            Divider()
                .overlay(.white)
                .frame(width: 320)
            
            HStack {
                Text(Constants.haveAccountText)
                    .foregroundStyle(Color.white)
                    .font(Font.custom("Roboto", size: 16))
            
                Button {
                    // TODO: IOS-15: Добавить логику обработки нажатий
                    print("[DEBUG]: Вход")
                } label: {
                    Text(Constants.singupButton)
                        .underline()
                        .foregroundColor(.editProfPurple)
                }
            }
        }
        .padding(.bottom, 130)
    }
```